### ISSUE TITLE
Survey issues and feedback

### ISSUE THREAD

**Issue by RichardTaylor** [mysociety/whatdotheyknow-theme#862]
User feedback:

>Sorry cant fill in your form.
>I have made several requests so I am not sure which one you refer to

This doesn't appear to be a completely unreasonable response given the text:

>We would like to know more about you, and your experience making an
> Freedom of Information request through the site.

Also I note "an Freedom of Information request" should be "a Freedom of Information request". 

This survey invite was sent to someone who has made many tens of requests over three years. 

cc @ajparsons

**Comment by ajparsons** [comment]
I've amended to the title description to tell people to answer with their most recent request in mind.

I've also adjusted the wording for some of the earlier questions to fit more easily for batch requests. 

The email template looks like it says:

>`Four weeks ago you made a Freedom of Information request to ' 
      '{{public_body_name}} using {{site_name}}.`

If we get more of this confusion, could include the request name here.

**Comment by garethrees** [comment]
> The email template looks like it says:
> 
> > `Four weeks ago you made a Freedom of Information request to '        '{{public_body_name}} using {{site_name}}.`

Presumably the `{{public_body_name}}` shouldn't be _outside_ of the quotes? Is that a typo here or a bug that needs fixing?

**Comment by RichardTaylor** [comment]
Further feedback:

>As we make many such requests we would need to know which one this refers to.  Perhaps you could include a url to the actual request.


>I've amended to the title description to tell people to answer with their most recent request in mind.

Presumably the outgoing email needs updating similarly?

**Comment by gbp** [comment]
> Presumably the `{{public_body_name}}` shouldn't be _outside_ of the quotes? Is that a typo here

Typo. There are no quotes in the app source or Transifex

**Comment by RichardTaylor** [comment]
A user has written:

>I've just received an email from you 'Whatdotheyknow' asking me to complete a survey.  I'm not sure this email is from you

The survey link is to a URL on the domain:

https://survey.alchemer.com

which isn't a survey domain I recognise .. it's not the typical surveymonkey or a Google form, and the URL contains lots of characters which might raise concerns.

Hosting the survey on the whatdotheyknow.com domain might improve confidence in it.

**Comment by ajparsons** [comment]
I'm blaming Alchemer for rebranding to a less clear name (but also if you google it is a survey platform). 

In the past, I have sometimes used a redirect under mysociety.org to tidy up long survey urls. Could explore something similar here, or possibly Alaveteli itself could redirect.

**Comment by garethrees** [comment]
> Could explore something similar here, or possibly Alaveteli itself could redirect.

I don't think we want to add the complexity to Alaveteli, and there's every chance we'd get the same correspondence after an internal redirect has happened.

> I'm blaming Alchemer for rebranding to a less clear name (but also if you google it is a survey platform).

Alchemer [supports custom DNS](https://help.alchemer.com/help/set-up-a-private-domain), so that could be worth a look in to, but I don't think this warrants any action unless we receive lots of correspondence along the same lines.

**Comment by RichardTaylor** [comment]
A perfectly reasonable response from a user to the survey invite, but one I thought I'd note here:

> No,
>Thank You.

**Comment by FOIMonkey** [comment]
User feedback: 

> survey is far too long.

**Comment by WilliamWDTK** [comment]
Is there a copy of the survey questions we can review?

**Comment by WilliamWDTK** [comment]
> User feedback:
> 
> > survey is far too long.

The same user has also said that:

> It took 7 minutes to do the survey.

that:

>that should be mentioned in the email.

And that there are:

>Some odd and probing questions in there.

**Comment by ajparsons** [comment]
Survey survey attached here:
[survey_5715101.pdf](https://github.com/mysociety/whatdotheyknow-theme/files/9564844/survey_5715101.pdf)

I'm fairly happy with the number of responses we are getting - 1,400 responses in the just over a year it's been up - which feels good for it's length (and is double the survey we replaced last year). Also happy to schedule in a review given it's now been up a year.

I'd like to work towards getting the same monthly reports going for this survey we have for a similar survey on the climate website.

**Comment by WilliamWDTK** [comment]
This is probably more about the email notification for the survey, but a user said that they didn't see why they needed to [complete it].

Our email just says it's to:

>it will help us make the site better.

That doesn't really provide much detail, but maybe its not practical to put more…

**Comment by HelenWDTK** [comment]
We've been contacted by a user who was confused when they got the survey link before a response was due from the authority, meaning there was no substantive response for them to give feedback on.

**Comment by garethrees** [comment]
> got the survey link before a response was due

The [survey alert](https://github.com/mysociety/alaveteli/blob/0.42.0.0/app/views/survey_mailer/survey_alert.text.erb) doesn't imply that a response from the authority is necessary to answer the survey, but I do recall that being a common comment with the previous iteration of the survey.

We do want to survey people who haven't had a response, but I agree that maybe waiting until after the due date would be better.

**Comment by ajparsons** [comment]
Didn't we push it back last year or is it still at 2 weeks? I've no objection to a longer wait.

**Comment by mdeuk** [comment]
+1

A user reported recently that they didn't know **which** request the survey pertained to, as they had made several requests, to the same body, within a similar period of time.

For ease of use, it would probably be helpful if a link to the request was in the email, even if we don't want to include it in the survey url itself.
